docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,354 @@
+- Enfore Layer 7 authorization policies using a waypoint proxy.
+- Split traffic between services using a waypoint proxy.
+- Gathering TCP telemetry and visualize all traffic between the pods.
+
+## Getting Started
+
+### Installation on OpenShift
+
+*Prerequisites*
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+
+*Steps*
+
+1. Install a Sail Operator using the CLI or through the web console. The steps can be found [here](https://github.com/istio-ecosystem/sail-operator/blob/main/docs/README.md#installation-on-openshift).
+
+2. Create the `istio-system` namespace and add a label `istio-discovery=enabled`.
+
+```bash
+$ kubectl create namespace istio-system
+$ kubectl label namespace istio-system istio-discovery=enabled
+```
+
+3. Create the `Istio` resource. 
+> **NOTE**:
+> The Istio resource `.spec.values.pilot.trustedZtunnelNamespace` value should match the namespace that we will install a `ZTunnel` resource at. 
+
+```bash
+$ cat <<EOF | kubectl apply -f-
+apiVersion: sailoperator.io/v1alpha1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.24.0
+  namespace: istio-system
+  updateStrategy:
+    type: InPlace
+  profile: ambient
+  values:
+    pilot:
+      trustedZtunnelNamespace: ztunnel
+    meshConfig:
+      discoverySelectors:
+        - matchLabels:
+            istio-discovery: enabled
+EOF
+```
+
+4. Confirm the installation and version of the control plane.
+
+```console
+$ kubectl get istio -n istio-system
+    NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+    default   1           1       0        default           Healthy   v1.24.0   23s
+```
+    Note: `IN USE` field shows as 0, as `Istio` has just been installed and there are no workloads using it.
+
+5. Create the `istio-cni` namespace.
+
+```bash
+$ kubectl create namespace istio-cni
+```
+
+6. Create the `IstioCNI` resource.
+
+```bash
+$ cat <<EOF | kubectl apply -f-
+apiVersion: sailoperator.io/v1alpha1
+kind: IstioCNI
+metadata:
+  name: default
+spec:
+  profile: ambient
+  version: v1.24.0
+  namespace: istio-cni
+EOF
+```
+
+7. Create the `ztunnel` namespace and add a label `istio-discovery=enabled`.
+> **NOTE**: 
+> We need to label both the `Istio` resource's namespace e.g. `istio-system` and the `ZTunnel` resource's namespace when using a `discoverySelectors` mesh config. Those two labels should be added before installing a `ZTunnel` instance. This approach is used to avoid a [TLS signing error](https://github.com/istio/istio/issues/52057).
+
+```bash
+$ kubectl create namespace ztunnel
+$ kubectl label namespace ztunnel istio-discovery=enabled
+```
+
+8. Create the `ZTunnel` resource.
+
+```bash
+$ cat <<EOF | kubectl apply -f-
+apiVersion: sailoperator.io/v1alpha1
+kind: ZTunnel
+metadata:
+  name: default
+spec:
+  profile: ambient
+  version: v1.24.0
+  namespace: ztunnel
+  values:
+    ztunnel:
+      image: docker.io/istio/ztunnel:1.24.0
```

Comment:
Probably we can remove this section as the default image pointing to the version will be used anyways.